bots/tests-invoke

```diff
@@ -351,6 +351,8 @@ class PullTask(object):
         elif prefix == "image" or prefix == "container":
             cmd = [ "timeout", "90m", os.path.join(test, "containers", "run-tests"),
                     "--container", value]
+        elif prefix == "cockpit":
+            cmd = [ "timeout", "90m", os.path.join(test, "run") ]
```

Comment:
This is a pretty clever intermediate solution that avoids large changes in tests-invoke. Nice!